### PR TITLE
484 box sizing border box (3)

### LINKED ISSUE #484: Add box-sizing border-box universal selector reset to styles.css
```
* {
  box-sizing: border-box;

  &::before,
  &::after {
    box-sizing: border-box;
  }
}
```

### PR DESCRIPTION
**Pull request by eduardseifert**
Fix #484

Added universal star selector for box-sizing: border-box (normalization, reset, streamline).

## Test URLs

- Before: https://main--hlx-test--urfuwo.hlx.page/design/design-system
- After: https://484-box-sizing-border-box-rebase--hlx-test--urfuwo.hlx.page/design/design-system (looks the same)

### CODE REVIEW

File: aemedge/blocks/contact/contact.css [hunk 1]
```diff
@@ -2,80 +2,82 @@
...
-  color: var(--udexColorGrey7);
-  background-color: var(--udexColorNeutralWhite);
-}
-
-.contact-wrapper .icon-sap-logo {
-  height: 32px;
-  width: 65.4px;
-  padding-right: var(--udexSpacer20);
+.contact-wrapper {
+  .logo-lang-picker {
```
Comment (alexcarol) [inline aemedge/blocks/contact/contact.css:6]: Do we have any policies regarding minimum support for css features? This feature seems to be supported in browsers with ~80% of the users (https://caniuse.com/css-nesting), is this enough for this site? /cc @saurabh-khare @mhaack

Comment (saurabh-khare) [reply aemedge/blocks/contact/contact.css:6]: @alexcarol 80% is a safe adaptability percentage. This means the rest 20% browsers are outdated, lacking security updates (not a probability for enterprise users) and do not support modern browser technology.

File: aemedge/blocks/footer/footer.css [hunk 2]
```diff
@@ -1,48 +1,51 @@
...
-}
-
-.block.footer a:hover {
-  color: var(--sapLink_Hover_Color);
-}
-
-footer .block.link-group p {
-  margin: 0;
+  padding-bottom: var(--udexSpacer40);
+  padding-top: var(--udexSpacer48);
```
Comment (saurabh-khare) [inline aemedge/blocks/footer/footer.css:5]: @eduardseifert Why do have changes in contact and footer files for this PR?

**Review by saurabh-khare — CHANGES_REQUESTED** [review]

File: aemedge/blocks/footer/footer.css (hunk 2 above)
Comment (eduardseifert) [reply aemedge/blocks/footer/footer.css:5]: 1. After introducing the `box-sizing: border-box` styles, the icons were not correctly rendered. I corrected this.
2. Took the possibility to nest `footer.css` and `contact.css` (and chore).

**Comment by saurabh-khare** [comment]
@eduardseifert I see problems on the content hub homepage
https://484-box-sizing-border-box-rebase--hlx-test--urfuwo.hlx.page/topics/
Screenshot 2024-04-23 at 09 17 33

In mobile view, the active marker on main nav is not aligned.
Screenshot 2024-04-23 at 09 27 46

**Comment by eduardseifert** [comment]
@saurabh-khare This is exactly the typical problem with `content-box`. What shall we (I) do? Again drive-by fixing?

**Comment by alexcarol** [comment]
Based on our last discussion, we'll fix the issues and proceed with merging so that we can take advantage of the new border-box setup.

**Comment by eduardseifert** [comment]
@saurabh-khare + @alexcarol As discussed, fixed the issues. Please take a look: https://484-box-sizing-border-box-rebase--hlx-test--urfuwo.hlx.page/topics/

**Review by saurabh-khare — APPROVED** [review]

**Comment by eduardseifert** [comment]
👊 💪

**Pull request merged**